src/Symfony/Component/DependencyInjection/Compiler/AutowireRequiredMethodsPass.php

```diff
@@ -85,7 +85,7 @@ protected function processValue(mixed $value, bool $isRoot = false): mixed
             $setters = $value->getMethodCalls();
             $value->setMethodCalls($withers);
             foreach ($setters as $call) {
-                $value->addMethodCall($call[0], $call[1], $call[2] ?? false);
+                $value->addMethodCall($call[0], $call[1], isset($call[2]) && $call[2]);
```

Comment:
I would not change that, as we expect the third array value to be a boolean. Instead, we should fix the invalid usages of the method calls.